src/_includes/components/social-icons.html

```diff
@@ -38,6 +38,13 @@ <h2 id="eslint-social-label" hidden>{{ site.footer.social_icons.title }}</h2>
 
             </a>
         </li>
-
+        <li>
+            <a href="{{ links.mastodon }}" rel="noopener noreferrer" target="_blank">
```

Comment:
```suggestion
            <a href="{{ links.mastodon }}" rel="me noopener noreferrer" target="_blank">
```

This brings the "verified" site links in a green box with a green checkmark to the mastodon account.

Like 
<img width="583" alt="image" src="https://user-images.githubusercontent.com/45708948/206909202-486da514-5054-40a0-bc41-c7eeded03ab3.png">

Reference <https://microformats.org/wiki/rel-me>

----

One more thing, I'm not sure if I should also add `rel="me"` for other links that point to ESLint accounts belonging to external sites. This may not be useful to them, but it would provide a degree of semanticization

Reply:
ya, let's see if we can add it or not.